Source/BSN.Commons.S3.Abstraction/BulkCredentialModel.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BSN.Commons.S3.Abstraction
+{
+    [Obsolete("[R.Noei] I'm not sure if we need this anymore.")]
+    public class BulkCredentialModel
+    {
+        public BulkCredentialModel(string endPoint,

```

Comment:
Add constructor comments, and use inhertidoc for param (such as SAF)

Reply:
I've added comment for it please review it.